tensorflow/compiler/xla/service/gpu/convolution_thunk.h

```diff
@@ -115,7 +115,9 @@ class ConvolutionThunk : public Thunk {
       perftools::gputools::dnn::ProfileResult* profile_result);
 
   // Returns the convolve algorithms that can be used for this ConvolutionThunk.
-  std::vector<perftools::gputools::dnn::AlgorithmType> GetAlgorithms(
+  // TODO(nluehr) GetAlgorithms should return AlgorithmDesc including both
```

Comment:
@jlebar. Sorry for not noting more clearly. In a future PR I will re-implement the loop over tensor ops inside GetAlgorithms, as noted in by this TODO.